config/config.yaml

```diff
@@ -318,105 +410,101 @@ clouds:
       # Hypershift Operator
       hypershift:
         image:
-          registry: quay.io
-          repository: acm-d/rhtap-hypershift-operator
           digest: sha256:0e6706e3bbc058a65f34dad7ccf048e56b1f32ca610c6f39f085ddb8bb1169ef
       # Backplane API
       backplaneAPI:
         image:
-          registry: arohcpsvcdev.azurecr.io
-          repository: app-sre/backplane-api
           digest: sha256:822477832a73c7eab7fe27200994f10030f708f4a752f33ded3f8f8eaa0470f6
       # Frontend
       frontend:
+        cert:
+          name: frontend-cert-{{ .ctx.regionShort }}
```

Comment:
`issuer: Self` surely has to remain in `dev` cloud but stuff like this name seems like a good candidate for promotion to top-level default

Reply:
done